src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final double defaultRadius = 10;
+    private final String defaultColor = "White";

```

Comment:
those should be constants, use constant case (uppercase with _ )
white - use Color.WHITE.name()